packages/dashboard/src/locales/i18n_generated.d.ts

```diff
@@ -0,0 +1,3 @@
+export function useDashboardI18N(): {}
```

Comment:
Hold on, this file should not be committed to git. Did I write `.gitignore` wrong?